tsconfig.json

```diff
@@ -22,8 +22,8 @@
       "@clr/angular": ["src/clr-angular"],
       "@clr/icons": ["src/clr-icons"],
       "@clr/icons/shapes/*": ["src/clr-icons/shapes", "src/clr-icons/shapes/*"],
-      "@clr/core": ["src/clr-core"],
-      "@clr/core/*": ["src/clr-core/*"]
+      "@clr/core": ["dist/clr-core"],
+      "@clr/core/*": ["dist/clr-core/*"]
```

Comment:
Always have projects point to the dist when using core. The ng9 PR has this same update.